configs/config.yaml

```diff
@@ -52,7 +52,7 @@ training:
   randomize_substeps: true
   segment_gradient_checkpointing: false
   learning_rate: 4e-3
-  resume_from_checkpoint: false
+  resume_from_checkpoint: true
```

Comment:
Does `resume_from_checkpoint: true` work when there is no checkpoints?

Reply:
It tries to find checkpoint in out folder first. If there is no such, then prints

`f"Didn't find a checkpoint in {training_args.output_dir}. Starting training from scratch"`

And starts from the scratch.